### DISCUSSION TITLE
Script Canvas nodes not appearing

### DISCUSSION THREAD

**Discussion by bearbones** [o3de/o3de#2410] [Help Wanted]
Haven't filed an issue for this because I don't know if it's reproducible.

Following the Script Canvas tutorial video, when I open Script Canvas, I can find the On Graph Start and Print nodes, but the rectangles next to their names in the Node Palette are transparent or gray instead of purple like in the video. When I drag them to the canvas, nothing appears.

![image](https://user-images.githubusercontent.com/57124166/126885799-93501aac-ac6b-4b4a-90eb-830e650eb97e.png)

Here is a log from Asset Processor containing warnings for Icons/ScriptCanvas/settings.png:
```
~~1627147429462~~1~~0000000000005BA8~~AssetBuilder~~AssetBuilder: Running processJob task
~~1627147429463~~1~~0000000000005BA8~~AssetBuilder~~AssetBuilder: Source = G:/GitHub/o3de/Assets/Editor/Icons/ScriptCanvas/settings.png
~~1627147429463~~1~~0000000000005BA8~~AssetBuilder~~AssetBuilder: Platform = pc
~~1627147429463~~1~~0000000000005BA8~~AssetBuilder~~Info: Performing image conversion: G:/GitHub/o3de/Assets/Editor/Icons/ScriptCanvas/settings.png
~~1627147429463~~1~~0000000000005BA8~~AssetBuilder~~W: Image Processing: Trace::Warning
~~1627147429463~~1~~0000000000005BA8~~AssetBuilder~~W: >	G:/GitHub/o3de/Gems/Atom/Asset/ImageProcessingAtom/Code/Source/BuilderSettings/BuilderSettingManager.cpp(524): 'struct AZ::Uuid __cdecl ImageProcessingAtom::BuilderSettingManager::GetSuggestedPreset(class AZStd::basic_string_view<char,struct AZStd::char_traits<char> >,class AZStd::shared_ptr<class ImageProcessingAtom::IImageObject>)'
~~1627147429463~~1~~0000000000005BA8~~AssetBuilder~~W: >	Image dimensions are not compatible with preset 'AlbedoWithGenericAlpha'. The default preset will be used.
~~1627147429473~~1~~0000000000005BA8~~AssetBuilder~~Image Processing: Image converted with preset [ReferenceImage] [G:/GitHub/o3de/Gems/Atom/Asset/ImageProcessingAtom/Config/ReferenceImage.preset] and saved to [G:/Projects/o3de/ccc/user/AssetProcessorTemp/JobTemp-wvDtwo] (784 bytes) taking 0.001000 seconds
~~1627147429473~~1~~0000000000005BA8~~AssetBuilder~~S: 0 errors, 1 warnings

```

Could I just be missing some files, and how can I check?

Edit: I clicked the UI button from this dropdown menu, and the node is at least visible, but something is still broken. ![image](https://user-images.githubusercontent.com/57124166/126886214-07f181e5-5543-4744-a146-2aae252c4db4.png)

**Comment by lsemp3d** [comment]
Sorry for the late reply, I have seen this issue before and it is because the ScriptCanvas stylesheet failed to load. If I remember correctly, the simplest solution was to delete the "Cache" folder inside of the O3DE folder (and possibly inside the project folder)
